### DISCUSSION TITLE
Elsa studio broken for sequence workflow

### DISCUSSION THREAD

**Discussion by stepankobzey** [elsa-workflows/elsa-core#5375] [Q&A] [answered]
Maybe someone had this elsa studio issue with custom activities. 
When creating a customer flowchart activity all the blocks just on top of each other and not spread around. which essentially not a big deal and can be moved. But when trying with sequence activity elsa studio display something weird. Its not showing subactivities and nothing can be added to the canvas. Running elsa studio from source shows same result!

Am I missing something?

![image](https://github.com/elsa-workflows/elsa-core/assets/3452761/974f3664-b711-45e6-a31d-254c03d8159a)

**Comment by glime-ai** [comment, marked as answer]
Based on the context provided, it seems you are encountering a known issue with Elsa Studio where the workflow designer does not currently support the Sequence activity. This is why you are seeing unexpected behavior when trying to create or display Sequence workflows in the designer. The activities are not displayed correctly, and you cannot add subactivities to the canvas.

This limitation has been acknowledged by the Elsa team, and there is an ongoing effort to add designer support for Sequence activities. Until this feature is implemented, you may experience the kind of issues you've described.

For the time being, you can continue to define Sequence workflows programmatically, but you won't be able to visualize them properly in Elsa Studio. Keep an eye on the Elsa GitHub repository for updates on this issue. The tracking of designer support for Sequence activities is mentioned in issue #3452 and #5200 on the Elsa GitHub issues page.

If you need further assistance or have more questions, you might want to reach out to the Elsa community or check for updates on the GitHub repository.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
